src/ProjectTemplates/test/Helpers/AspNetProcess.cs

```diff
@@ -111,6 +113,36 @@ private Uri GetListeningUri(ITestOutputHelper output)
             }
         }
 
+        public async Task AssertLinksWork(string requestUrl)
+        {
+            var request = new HttpRequestMessage(
+                HttpMethod.Get,
+                new Uri(_listeningUri, requestUrl));
+
+            var response = await _httpClient.SendAsync(request);
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var parser = new HtmlParser();
+            var html = await parser.ParseAsync(await response.Content.ReadAsStreamAsync());
+            foreach (IHtmlAnchorElement link in html.Links)
+            {
+                if(link.Protocol == "about:")
```

Comment:
What is this for?

Reply:
AngleSharp interprets our relative links as having Protocol "about:", I'm just using that to behave differently for absolute and relative links.